src/main/java/org/reaktivity/nukleus/oauth/internal/Capabilities.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright 2016-2019 The Reaktivity Project
+ *
+ * The Reaktivity Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package org.reaktivity.nukleus.oauth.internal;
+
+public class Capabilities
+{
+    private static final int CHALLENGE_MASK = 0x01;
+    private static final int CHALLENGE_BIT = 1;
+
+    public static boolean canChallenge(
+        int capabilities)
+    {
+        return (capabilities & CHALLENGE_MASK) == CHALLENGE_BIT;
```

Comment:
This happens to work because both `CHALLENGE_MASK` and `CHALLENGE_BIT` are `1`, but this should really be changed to:
```
return (capabilities & CHALLENGE_MASK) != 0;
```
... and then remove `CHALLENGE_BIT`.